Dockerfile.concourse

```
@@ -1,4 +1,4 @@
-FROM onsdigital/dp-concourse-tools-ubuntu
+FROM onsdigital/dp-concourse-tools-ubuntu:ubuntu20.4-rc.1
```

Due to creating a new folder ubuntu-20 in dp-concourse-tools, the repository in docker is actually onsdigital/dp-concourse-tools-ubuntu-20 and hence docker image is onsdigital/dp-concourse-tools-ubuntu-20:ubuntu20.4-rc.1:ubuntu20.4-rc.1

Either we re-upload this under the onsdigital/dp-concourse-tools-ubuntu docker repository or update the reference here

```suggestion
FROM onsdigital/dp-concourse-tools-ubuntu-20:ubuntu20.4-rc.1
```

---

Let me know what you would prefer, I can always upload another image and delete the docker repo referring to ubuntu-20